.rubocop.yml

```diff
@@ -11,13 +11,18 @@ AllCops:
     - "db/migrate/*good_job*.rb"
     - "scripts/screenshots/**/*"
   NewCops: enable
-  TargetRubyVersion: 3.3.7
+  TargetRubyVersion: 3.4.2
   SuggestExtensions: false
 
 Bundler/OrderedGems:
   Exclude:
     - 'Gemfile'
 
+Gemspec/RequiredRubyVersion:
+  Exclude:
+    - 'lib/anonymizer/anonymizer.gemspec'
+    - 'lib/omniauth-rdv-service-public/omniauth-rdv-service-public.gemspec'
+
```

Comment:
Rubocop remontait des offenses comme ça: 

lib/anonymizer/anonymizer.gemspec:10:32: W: Gemspec/RequiredRubyVersion: required_ruby_version and TargetRubyVersion (3.4, which may be specified in .rubocop.yml) should be equal.

Je pense que cette cop est destinée aux librairies qui n'ont qu'un seul fichier gemspec, mais pas à notre cas de gem inclue dans le repo.